src/main/java/io/stargate/sgv2/jsonapi/service/resolver/CreateKeyspaceCommandResolver.java

```diff
@@ -0,0 +1,73 @@
+package io.stargate.sgv2.jsonapi.service.resolver;
+
+import io.stargate.sgv2.jsonapi.api.model.command.CommandContext;
+import io.stargate.sgv2.jsonapi.api.model.command.impl.CreateKeyspaceCommand;
+import io.stargate.sgv2.jsonapi.service.cqldriver.executor.DatabaseSchemaObject;
+import io.stargate.sgv2.jsonapi.service.operation.Operation;
+import io.stargate.sgv2.jsonapi.service.operation.keyspaces.CreateKeyspaceOperation;
+import jakarta.enterprise.context.ApplicationScoped;
+import java.util.Map;
+
+/**
+ * Command resolver for {@link CreateKeyspaceCommand}. Responsible for creating the replication map.
+ * Resolve a {@link CreateKeyspaceCommand} to a {@link CreateKeyspaceOperation}
+ */
+@ApplicationScoped
+public class CreateKeyspaceCommandResolver implements CommandResolver<CreateKeyspaceCommand> {
```

Comment:
-1 for duplicating code like this, create  a common subclass for create Keyspace and create Namespace